src/Schema/AbstractSchemaManager.php

```
@@ -156,6 +156,7 @@ public function listSequences($database = null)
     {
         if ($database === null) {
             $database = $this->_conn->getDatabase();
+            assert($database !== null);
```

I don't know how likely it is these assertions will actually fail. If there is a more appropriate exception I should throw, let me know.

---

Probably, it should be an exception. Let’s suppress all errors with non-obvious fixes instead of fixing here.